slicker/cleanup.py

```diff
@@ -10,9 +10,8 @@
 import ast
 import difflib
 import os
-import sys
 
-from fix_includes import fix_python_imports
+from isort import SortImports
```

Comment:
This should be `import isort`, then `isort.SortImports` below, to conform to [Khan's python style](https://github.com/Khan/style-guides/blob/master/style/python.md#imports).

Reply:
Ok, I'll change that

Reply:
I made a fixup for that